src/components/bulkConcepts/component/ActionButtons.jsx

```diff
@@ -69,6 +71,7 @@ export class ActionButtons extends Component {
             this.props.preview.display_name,
           )
           }
+          disabled={this.state.disableBtn}
```

Comment:
@ozezzy, I think it is best to destructure all props and states;

Reply:
done